### PR TITLE
tests for ruby [gem] service

### PR DESCRIPTION
**Pull request by chris48s**

### CODE REVIEW

File: service-tests/gem.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const {
+  isVPlusDottedVersionAtLeastOne,
+  isMetric
+} = require('./helpers/validators');
+const isOrdinalNumber = Joi.string().regex(/^[0-9]+(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ)$/);
+const isOrdinalNumberDaily = Joi.string().regex(/^[0-9]+(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ) daily$/);
```
Comment (paulmelnikow) [inline service-tests/gem.js]: How about we exclude 0th, which is probably an error and not an ordinal?

File: service-tests/gem.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  .expectJSONTypes(Joi.object().keys({
+    name: 'downloads',
+    value: Joi.string().regex(/^[1-9][0-9]*[kMGTPEZY]? stable version$/)
+  }));
+
+t.create('version downloads (valid, specific version)')
+  .get('/dv/rails/4.1.0.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'downloads',
+    value: Joi.string().regex(/^[1-9][0-9]*[kMGTPEZY]? version 4.1.0$/)
```
Comment (paulmelnikow) [inline service-tests/gem.js]: I think these should be changed to **downloads@4.1.0** and **downloads@stable**. That’s similar to the **npm@next** badge. Would you be up for changing them?

**Comment by chris48s** [comment]
updated as requested

**Comment by paulmelnikow** [comment]
Sweet, thanks!

**Pull request merged**